recipes-wpe/wpeframework/include/cobalt.inc

```diff
@@ -19,7 +19,3 @@ PACKAGECONFIG[cobalt] = " \
     ${WPE_COBALT_EXTRAFLAGS}, \
     -DPLUGIN_COBALT=OFF,libcobalt,libcobalt \
 "
-# ----------------------------------------------------------------------------
-
-DEPENDS += " libcobalt"
-RDEPENDS_${PN} += " libcobalt"
```

Comment:
Can not remove it from the dependency, can you make it as the conditional based ?

Reply:
Please note that the dependencies already established via PACKAGECONFIG (when "cobalt" is enabled), so these extra DEPENDS and RDEPENDS should be not only unnecessary as also cause trouble when we don't want cobalt enabled...

You're probably needing it because the PACKAGECONFIG for cobalt also has an issue, which I also fix in this PR in commit https://github.com/WebPlatformForEmbedded/meta-wpe/pull/403/commits/1d3a8c253c0c2538d7367d0c6e629f5fea848378.

Reply:
Ok :)